benchmarks/README.md

```diff
@@ -61,11 +61,31 @@ python benchmark_serving.py \
 
 ```
 
+## Benchmark with openorca dataset (openorca is used by MLPerf inference for LLaMA2 models)
+```
+python JetStream/benchmarks/benchmark_serving.py   \
+--tokenizer ~/maxtext/assets/tokenizer.llama2  \
+--warmup-first true   \
+--save-result   \
+--save-request-outputs   \
+--request-outputs-file-path outputs.json   \
+--num-prompts 1000   \
+--max-output-length 1024   \
+--dataset openorca
+
+```
+
 ## Standalone Evaluation Run
 
 If you used `--save-request-outputs`, you can separately evaluate against the generated outputs.
 
 ```
 python eval_accuracy.py
```

Comment:
Lets put this as "python eval_accuracy.py output.json" to be consistent with above command.

Reply:
done